qiita_db/metadata_template/base_metadata_template.py

```diff
@@ -585,6 +588,82 @@ def _check_special_columns(cls, md_template, obj):
             cls._check_template_special_columns(md_template, obj))
 
     @classmethod
+    def _clean_validate_template(cls, md_template, study_id, obj,
```

Comment:
Can we get another dev to chime in on this? It's a little disconcerting to me to just take it on faith the study is correct and wait until it fails to see otherwise.